src/styles/common.scss

```diff
@@ -90,6 +90,28 @@
     color: var(--primary1);
 }
 
+/* Input **********************************************************************/
+
+input.text-input {
+    color: var(--primary5);
+    background: transparent;
+    padding: 8px 0;
+    border-bottom: solid 1px rgba(23, 17, 13, 0.4);
+}
+
+input.text-input {
+    box-shadow: none;
+}
+
+input.text-input:disabled {
```

Comment:
how does the placeholder text look like? maybe it should be styled it as well?

Reply:
in wix template there is no placeholders:
![Screenshot 2024-11-05 at 12 56 33](https://github.com/user-attachments/assets/8cf6c346-e447-4dfe-98e4-3ebabfb3a90b)

so I'm not sure what is expected in this case

Reply:
anyway, added some styles for placeholder and added this state to a board